index.js

```diff
@@ -1,8 +1,22 @@
 const config = {
+    env: {
```

Comment:
I think `env` rules should be defined by the project not the default ruleset. Ideally we would have a local `.eslintconfig` that extends these rules where we apply the environment specific stuff.

Reply:
You know, you're probably right about that. I'm just trying to make a replicable config that works across our monorepo and other projects in the same stack and ecosystem. But maybe `env` is a bridge too far.